api/metrics-api-jaxrs-1.1/src/main/java/org/hawkular/metrics/api/jaxrs/config/ConfigurationKey.java

```diff
@@ -26,6 +26,7 @@
  */
 public enum ConfigurationKey {
 
+    ALLOWED_CORS_ORIGINS("hawkular.metrics.allowed-cors-origins", "*", "ALLOWED_CORS_ORIGINS", false),
```

Comment:
Shouldn't we use an empty string by default (`deny all` instead of `allow any`)?